packages/eslint-plugin-pf-codemods/lib/helpers.js

```diff
@@ -186,7 +211,12 @@ function getPackageImports(context, packageName, importNames = []) {
   const specifiers = context
     .getSourceCode()
     .ast.body.filter((node) => node.type === "ImportDeclaration")
-    .filter((node) => node.source.value === packageName)
+    .filter((node) => {
+      if(packageName.startsWith("@patternfly")) {
+        return pfPackageMatches(packageName, node.source.value);
+      }
+      return node.source.value === packageName
+    })
```

Comment:
Do we need this `if` block to return one of two things here considering other helpers just use `pfPackageMatches` without any sort of `startsWith` check?

Reply:
I wouldn't block over making it a `packageMatches` vs `pfPackageMatches` helper. We can always cross that bridge when we come to it if anything, but could make sense to try and open it up beyond just patternfly now.

Any comment re: the `charts-` files?

Reply:
@thatblindgeye  - just pushed with a couple `charts` tests. also updated the matcher to check `dist/esm` or `dist/js`

Reply:
as far as opening beyond just pf... i guess i'm not familiar enough with this to know that other packages would have similar structure with esm/js exports, @thatblindgeye